packages/core/chrome/core-chrome-browser-internal/src/ui/header/header_help_menu.tsx

```diff
@@ -162,7 +162,7 @@ export class HeaderHelpMenu extends Component<Props, State> {
         <EuiButtonEmpty href={helpSupportUrl} target="_blank" size="s" flush="left">
           <FormattedMessage
             id="core.ui.chrome.headerGlobalNav.helpMenuAskElasticTitle"
-            defaultMessage="Ask Elastic"
+            defaultMessage="Ask Elastic Support"
```

Comment:
This will change the label for all distributions, including on-prem. I'm afraid addressing this issue won't be that simple. 

ATM the custom support url for Cloud is set here:

https://github.com/elastic/kibana/blob/46ccdc9ee012cadfd30127719d1bf11dfd375637/x-pack/plugins/cloud/public/plugin.tsx#L144-L145

I guess we would need to change the API to additionally allow to change the label associated with the link...

BTW @afharo, the `cloud` plugin is included in all our distributions, right? And I'm not seeing anything disabling this plugin by default in it's config, so it would mean that this `setHelpSupportUrl` call during the `setup` of the cloud plugin is always executed atm (even on-prem?)

Reply:
> This will change the label for all distributions, including on-prem.

Is it too bad to have that common wording?

Reply:
> it would mean that this setHelpSupportUrl call during the setup of the cloud plugin is always executed atm (even on-prem?)

That seems accurate to me... I wonder when the regression was introduced...

Reply:
> Is it too bad to have that common wording?

Fair question. if docs and/or product think it's not, then it's probably fine